recipes/sassc/all/conanfile.py

```diff
@@ -67,25 +69,34 @@ def _build_msbuild(self):
 
     def build(self):
         self._patch_sources()
-        with tools.chdir(self._source_subfolder):
-            if self._is_msvc:
+        with chdir(self, self._source_subfolder):
+            if is_msvc(self):
                 self._build_msbuild()
             else:
                 self.run("{} -fiv".format(tools.get_env("AUTORECONF")), run_environment=True)
-                tools.save(path="VERSION", content="%s" % self.version)
+                save(self, path="VERSION", content=f"{self.version}")
                 autotools = self._configure_autotools()
                 autotools.make()
 
     def package(self):
-        with tools.chdir(self._source_subfolder):
-            if self._is_msvc:
+        with chdir(self, self._source_subfolder):
+            if is_msvc(self):
                 self.copy("*.exe", dst="bin", src=os.path.join(self._source_subfolder, "bin"), keep_path=False)
             else:
                 autotools = self._configure_autotools()
                 autotools.install()
         self.copy("LICENSE", src=self._source_subfolder, dst="licenses")
 
     def package_info(self):
-        bin_path = os.path.join(self.package_folder, "bin")
-        self.output.info("Appending PATH env var with : {}".format(bin_path))
-        self.env_info.PATH.append(bin_path)
+        self.cpp_info.frameworkdirs = []
+        self.cpp_info.libdirs = []
+        self.cpp_info.resdirs = []
+        self.cpp_info.includedirs = []
+
+        bin_folder = os.path.join(self.package_folder, "bin")
+        # In case need to find packaged tools when building a package
+        self.buildenv_info.append("PATH", bin_folder)
+        # In case need to find packaged tools at runtime
+        self.runenv_info.append("PATH", bin_folder)
```

Comment:
```suggestion
```
In the past I thought it was needed, but after reading docs and checking those generated scripts, PATH is automatically filled by Conan: 

https://docs.conan.io/en/latest/reference/conanfile/tools/env/virtualbuildenv.html
https://docs.conan.io/en/latest/reference/conanfile/tools/env/virtualrunenv.html